packages/visualizations-react/stories/Table/data.ts

```diff
@@ -72,21 +80,16 @@ export default [
       url: "https://example.com/integer-id"
     },
     {
-      title: "Nam Ullamcorper Blog Post",
-      content: "Nam ullamcorper semper lacus, ac hendrerit leo tempor nec. Nunc id urna a urna scelerisque viverra sit amet a felis.",
-      datePublished: "2024-02-20",
-      isFeatured: true,
-      wordCount: 1400,
-      readingTime: 6.5,
-      url: "https://example.com/nam-ullamcorper"
+      title: "Undefined row",
+      price: undefined,
+      content: undefined,
+      datePublished: undefined,
+      isFeatured: undefined,
+      wordCount: undefined,
+      readingTime: undefined,
+      url: undefined
```

Comment:
For TS and completness sake we could add one with `null` as well (or just mix some keys as `null`). The API is more likely to output `null` by default (although, it can be changed by the user).

Reply:
Done